src/Modules/Backend/templates/base/head.html.twig

```diff
@@ -13,11 +13,11 @@
 
   {{ encore_entry_script_tags('Backend/js/Backend', null, 'BackendConfig') }}
   {{ encore_entry_link_tags('Backend/css/screen', null, 'BackendConfig') }}
-  {% for jsFile in jsFiles %}
-    <script src="/{{ jsFile }}"></script>
+  {% for js_file in jsFiles %}
+    <script src="/{{ js_file }}"></script>
   {% endfor %}
-  {% for cssFile in cssFiles %}
-    <link href="/{{ cssFile }}" rel="stylesheet">
+  {% for css_file in cssFiles %}
```

Comment:
Shouldn't `cssFiles` be changed to `css_files`?

Reply:
fixed 2589ead44a4e5286af6b0d8ad54ddd08d28b77f1